lib/ex_admin/param_associations.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ExAdmin.ParamsAssociations do
+  @moduledoc false
+  require Logger
+  import Ecto.Query
+
+  def load_associations(params, model_name, model) do
+    case Map.has_key? params, model_name do
+      true -> Map.put(params, model_name, load_associations(params[model_name], model))
+      false -> params
+    end
+  end
+
+  defp load_associations(params, model) do
+    Enum.reduce(Map.keys(params), params, fn(key, p) ->
+      key_as_string = Atom.to_string(key)
+      cond do
+        String.ends_with?(key_as_string, "attributes") ->
+          new_key = String.replace_suffix(key_as_string, "_attributes", "")
+            |> String.to_atom
+          {_, p} = Map.pop(p, key)
+          value = remove_destroyed_associations(params[key])
+          Map.put(p, new_key, value)
+				String.ends_with?(key_as_string, "_ids") ->
```

Comment:
Gah, I some time turn on set paste in vim and forget to turn it off. Then chaos occurs.